lms/templates/courseware/courseware-chromeless.html

```diff
@@ -201,4 +201,145 @@
     }
   }());
 </script>
+<%text>
+<script type="text/javascript">
+  /**
+   * @description Overrides AnnotatorJS default resizing behavior with a custom resizer. 
+   *              Automatically intercepts new .annotator-resize elements and applies 
+   *              a more direct, mouse-based resizing for <textarea> elements.
+   */
+
+  (() => {
+    /**
+     * Returns true if the Annotatorjs Editor is displayed downward.
+     *
+     * @param {HTMLElement} resizer - The element (or a descendant) within the editor.
+     * @returns {boolean}
+     */
+    const isEditorDownward = (resizer) =>
+      resizer.closest('.annotator-editor')?.classList.contains('annotator-invert-y') || false;  
+
+    /**
+     * A class that makes a <textarea> resizable by clicking and dragging an element.
+     */
+    class ResizableTextarea {
+      /**
+       * @param {HTMLElement} draggable - The element that initiates the resize on mousedown.
+       * @param {HTMLTextAreaElement} textarea - The <textarea> to be resized.
+       */
+      constructor(draggable, textarea) {
+        this.draggable = draggable;
+        this.textarea = textarea;
+        this.startX = 0;
+        this.startY = 0;
+        this.startWidth = 0;
+        this.startHeight = 0;
+
+        this.onResize = this.onResize.bind(this);
+        this.stopResize = this.stopResize.bind(this);
```

Comment:
This is misleading, please avoid calling attributes and methods with the same name. @Serafin-dev